src/sentry/templates/sentry/integrations/gitlab-config.html

```diff
@@ -49,7 +49,7 @@ <h3>{% trans "Step 2: Connect Sentry with your App" %}</h3>
 {% else %}
 <h3>{% trans "Step 1: Create a Sentry App in GitLab" %}</h3>
   <p>{% trans "To configure GitLab with Sentry, you will need to create a Sentry app in your GitLab instance." %}</p>
-  <p>{% trans "You'll also need to be a maintainer or owner in GitLab." %}</p>
+  <p>{% trans "You'll also need to be a maintainer or owner in GitLab. Projects owned users are not supported." %}</p>
```

Comment:
Does this sentence read right?

Seems like this should say `Projects owned by users are not supported`.

Reply:
derp.

Reply:
gotta pwn those users.